Unable to compile programs that use libfxcg functions in them.

I've compiled some basic hello work programs and such, but was interested in drawing some sprites to the screen, so i defined by sprite data in code and tried to use `VRAM_CopySprite` to load up the sprite but i just get the error `undefined reference to _VRAM_CopySprite`. I've checked over my makefile (It's the default one) and it seems the library is getting included fine and i'm doing `#include <fxcg/display.h>` so I don't see why this issue is being caused. I would attach my full code but honestly not sure it's necessary, i'm just calling `VRAM_CopySprite(logo, 249, 25, 80, 58);` where `logo` is a whole bunch of color_t data.

Forgive me if this is basic or not appropriate for an issue, I'm new to github _and_ not really familiar with build systems all that much since my background is in python.

Thanks in advance, Evan.

--- a/examples/skeleton/Makefile
+++ b/examples/skeleton/Makefile
@@ -31,7 +31,11 @@ INCLUDES	:=
 
 MKG3AFLAGS := -n basic:example -i uns:../unselected.bmp -i sel:../selected.bmp
 
-CFLAGS	= -Os -Wall $(MACHDEP) $(INCLUDE) -ffunction-sections -fdata-sections 
+# Optional: add -flto to CFLAGS and LDFLAGS to enable link-time optimization
+# (LTO). Doing so will usually allow the compiler to generate much better code
+# (smaller and/or faster), but may expose bugs in your code that don't cause
+# any trouble without LTO enabled.
+CFLAGS	= -Os -Wall $(MACHDEP) $(INCLUDE) -ffunction-sections -fdata-sections
 CXXFLAGS	=	$(CFLAGS) -fno-exceptions
 
 LDFLAGS	= $(MACHDEP) -T$(FXCGSDK)/toolchain/prizm.x -Wl,-static -Wl,-gc-sections

--- a/libc/Makefile
+++ b/libc/Makefile
@@ -1,8 +1,8 @@
 TOOLCHAIN_PREFIX=sh3eb-elf-
 CC=$(TOOLCHAIN_PREFIX)gcc
 AS=$(TOOLCHAIN_PREFIX)as
-AR=$(TOOLCHAIN_PREFIX)ar
-CFLAGS=-c -ffunction-sections -fdata-sections -Os \
+AR=$(TOOLCHAIN_PREFIX)gcc-ar
+CFLAGS=-c -ffunction-sections -fdata-sections -Os -flto -ffat-lto-objects \
 	   -m4a-nofpu -mhitachi -ffreestanding -Wall -Wsystem-headers -g -I../include
 LDFLAGS=-Wl,-static -lfxcg
 ARFLAGS=rsv

--- a/libfxcg/Makefile
+++ b/libfxcg/Makefile
@@ -1,9 +1,9 @@
 TOOLCHAIN_PREFIX=sh3eb-elf-
 CC=$(TOOLCHAIN_PREFIX)gcc
 AS=$(TOOLCHAIN_PREFIX)as
-AR=$(TOOLCHAIN_PREFIX)ar
+AR=$(TOOLCHAIN_PREFIX)gcc-ar
 CFLAGS=-c -ffunction-sections -fdata-sections -Os -Lr -I../include \
-	   -mhitachi -m4a-nofpu -flto -std=c99 -Wall -Wextra
+	   -mhitachi -m4a-nofpu -flto -ffat-lto-objects -std=c99 -Wall -Wextra
 ARFLAGS=rs
 VPATH=syscalls
 SHSOURCES=$(wildcard syscalls/*.S) $(wildcard misc/*.S)

--- a/toolchain/prizm_rules
+++ b/toolchain/prizm_rules
@@ -20,7 +20,7 @@ PREFIX	:=	sh3eb-elf-
 export AS	:=	$(PREFIX)as
 export CC	:=	$(PREFIX)gcc
 export CXX	:=	$(PREFIX)g++
-export AR	:=	$(PREFIX)ar
+export AR	:=	$(PREFIX)gcc-ar
 export OBJCOPY	:=	$(PREFIX)objcopy
 export MKG3A := mkg3a
 

